src/pages/createCohort/components/cohortList.jsx

```diff
@@ -17,20 +24,67 @@ const CohortList = () => {
       .catch(console.log);
   }, []);
 
+  const updateCohortName=(event)=>{
+    event.preventDefault()
+    if (!newCohortName){setError(true); setTimeout(() => {
+      setError(false)
+    }, 3000);}
+    else{
+    client.patch(`/cohort/${event.target.id}`,{name:newCohortName},true).then(res=>{if (res.data.status === 'success') {
+      setUpdateCohortRes(true);
+    }}).catch(console.log)
```

Comment:
This is quite difficult to read, please format this a little nicer so it's easier to tell what the if statements and callback functions are related to